.circleci/config.yml

```diff
@@ -1,22 +1,176 @@
+# Python CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+#
 ---
-version: 2
+version: 2.1
+executors:
+    # here we can define an executor that will be shared across different jobs
+    python-executor:
+        docker:
+            - image: python:3.7-slim-buster
+              environment:
+                  POETRY_CACHE: /work/.cache/poetry
+                  PIP_CACHE_DIR: /work/.cache/pip
+                  JUPYTER_RUNTIME_DIR: /work/.cache/jupyter/runtime
+                  JUPYTER_CONFIG_DIR: /work/.cache/jupyter/config
+                  SHELL: bash -l
+        working_directory: /work
+        resource_class: large
+    docker-executor:
+        docker:
+            - image: dreamquark/docker:latest
```

Comment:
just to understand: is this where we name our executor or where we pull or image? What's specific with dreamquark/docker:latest image?

Reply:
So we name the executor, with image name and so on, so we don't have to say each time which image use, where is the work dir and so on.

Dreamquark image is basically docker one, but installed with Make, and some other usefuls tools.
I can change this if you want in another PR, but then I need to install Make and so on. 
I think this is a bit faster, but I can change